doc/man1/flux-shutdown.rst

```diff
@@ -55,6 +60,22 @@ OPTIONS
    Increase output verbosity.  Level 1 shows all log messages.  Higher
    verbosity levels are reserved for future use.
 
+**--dump=PATH**
+   Dump a checkpoint of KVS content to *PATH* using :man1:`flux-dump` after the
+   KVS has been unloaded.  The dump may be restored into a new Flux instance
+   using :man1:`flux-restore`.  Dump creation adds time to the shutdown
+   sequence, proportional to the amount of data in the KVS.  "auto" is a
+   reserved *PATH* for KVS garbage collection (see below).
```

Comment:
instead of "see below", perhaps say equivalent to `--gc`.  Not super clear it's referring to `--gc` text.